onyo/main.py

```diff
@@ -550,6 +550,53 @@ def setup_parser():
         type=path,
         help='assets or directories for which to set values'
     )
+    #
+    # subcommand "get"
+    #
+    cmd_get = subcmds.add_parser(
+        'get',
+        description=textwrap.dedent(commands.get.__doc__),
+        formatter_class=SubcommandHelpFormatter,
+        help='print the requested key(s) for matching assets'
+    )
+    cmd_get.set_defaults(run=commands.get)
+    cmd_get.add_argument(
+        '-d', '--depth',
+        metavar='DEPTH',
+        type=int,
```

Comment:
If I had to guess, this is where the shell completion is exploding. IIRC, all types need to be listed in that command, and it doesn't know about integers or strings yet.